open-api.yaml

```diff
@@ -1263,16 +1287,18 @@ paths:
                   q: Harry
                   offset: 0
                   limit: 20
-                  filter: '("genres:Horror" OR "genres:Mystery") AND "director:Jordan Peele"'
+                  filter: (genres = Horror AND genres = Mystery) OR release_date > 523242000
                   facetsDistribution:
-                    - string
+                    - genres
+                    - author
```

Comment:
Should the examples not be formatted in JSON?

`'["genres", "author"`]'`

Not sure! Because this is also confusing as it looks like as a string more than a JSON

Reply:
This is automatically generated by stoplight as the open api specification. It seems that it uses a different syntax for a parameter example and a body request example.